template/src/components/virtual-background/useVB.tsx

```diff
@@ -69,6 +76,8 @@ type VBContextValue = {
   setSaveVB: React.Dispatch<React.SetStateAction<boolean>>;
```

Comment:
update to saveVirtualBackground

Reply:
this is for core , have updated methods in customization vb api